frontend/src/packages/auth/auth-api.ts

```diff
@@ -32,7 +32,10 @@ class AuthApi extends BaseHttpApi {
         hasAuth: false,
       },
     );
-
+    if (!response.ok) {
+      const error = await response.json<{ code: number; message: string }>();
+      throw new Error(error.message);
+    }
```

Comment:
As for this one too https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/44